test-suite/src/test/scala/org/scalajs/testsuite/javalib/LinkedListTest.scala

```diff
@@ -0,0 +1,357 @@
+/*                     __                                               *\
+**     ________ ___   / /  ___      __ ____  Scala.js Test Suite        **
+**    / __/ __// _ | / /  / _ | __ / // __/  (c) 2013-2015, LAMP/EPFL   **
+**  __\ \/ /__/ __ |/ /__/ __ |/_// /_\ \    http://scala-js.org/       **
+** /____/\___/_/ |_/____/_/ | |__/ /____/                               **
+**                          |/____/                                     **
+\*                                                                      */
+package org.scalajs.testsuite.javalib
+
+import scala.language.implicitConversions
+
+import scala.collection.JavaConversions._
+import scala.collection.mutable
+
+import org.scalajs.jasminetest.JasmineTest
+
+import scala.scalajs.runtime.UndefinedBehaviorError
+
+import java.util.LinkedList
+import java.util.Comparator
+
+object LinkedListTest extends JasmineTest {
+
```

Comment:
It is not so much work to copy, paste and fix results as per collection peculiarity.
At this point I have to work a bit to really refactor the tests in a good shape, I also want to move all this stuffs in another package.

Let say I will do it before 0.6.4 will be released.